katas/content/superdense_coding/index.md

```diff
@@ -0,0 +1,30 @@
+# Superdense Coding Kata
```

Comment:
When you create a new kata, you need to add it to index.json - otherwise CI build will not build it. (Then, if it's still a work in progress, you remove the addition from the list before merging the PR, so that it doesn't get published accidentally.)

Reply:
Done. I tested it locally before pushing :-)

I was wondering if its worth having a WIP list of katas that get tested in CI and local but not get published. That will give us immunity from accidental publishing :-)